README.md

```diff
@@ -19,6 +19,8 @@ It currently provides five different interfaces to serve your emulation needs, a
 ## [Installation](#installation)
 To install `flare-emu` for IDA Pro, simply drop `flare_emu.py`, `flare_emu_ida.py`, and `flare_emu_hooks.py` into your IDA Pro's `python` directory and import it as a module in your IDAPython scripts. 
 
+To install `flare-emu` for Rizin , simply ensure that `flare_emu.py`, `flare_emu_rizin.py`, and `flare_emu_hooks.py` are in Python's search path for importing modules. When using Radare2 as the binary analysis component for `flare-emu`, [rzpipe](https://github.com/rizinorg/rz-pipe) is required.
```

Comment:
should "Radare2" be "Rizin" here? copy/paste error?

Reply:
Fixed!